src/components/expression/expression-renderer.js

```diff
@@ -217,54 +217,41 @@ genotet.ExpressionRenderer.ZoomStatus = function(params) {
     params.conditionNames : [];
 };
 
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_HEIGHT_PERCENTAGE = 0.2;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_TFA_PROFILE_HEIGHT_PERCENTAGE =
+/** @private @const {number} */
+genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_HEIGHT_PERCENTAGE_ = 0.2;
+/** @private @const {number} */
+genotet.ExpressionRenderer.prototype.DEFAULT_TFA_PROFILE_HEIGHT_PERCENTAGE_ =
   0.2;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_LEGEND_HEIGHT = 25;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_LEGEND_MARGIN = 10;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_MARGIN = 40;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_CIRCLE_SIZE = 1;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_HEATMAP_GRADIENT_HEIGHT = 20;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.GENE_LABEL_WIDTH_FACTOR = 8.725;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.CONDITION_LABEL_HEIGHT_FACTOR = 6.501175;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.LABEL_MARGIN = 10;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.LABEL_DIFFERENCE = 10;
-
-/** @const {number} */
-genotet.ExpressionRenderer.prototype.HEATMAP_LEGEND_MARGIN = 1;
-
-/** @const {number} */
+/** @private @const {number} */
+genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_LEGEND_HEIGHT_ = 25;
```

Comment:
If you are not planning for custom legend font size (which I personally think is not necessary), DEFAULT can also be saved here.